pkg/apis/eventing/v1alpha1/trigger_types_test.go

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var triggerConditionReady = duckv1alpha1.Condition{
```

Comment:
Put all of these in a single `var` block:

```go
var(
  triggerConditionReady = ...
  triggerConditionBrokerExists = ...
  ...
)
```

Reply:
Done